tests/test_features.py

```diff
@@ -10,14 +15,6 @@
 except KeyError:
     pass
 
-import matplotlib
```

Comment:
> Just to be clear, none of the previous normalization behavior has changed, right?

Right.

> Otherwise, I think we're in good shape. There are still some unresolved line comments (docstrings mainly) that I'd like to see resolved before merging.

Did a double-check. Fixed all your comments. Saw anything else?